cns/imdsclient/api.go

```diff
@@ -74,4 +74,5 @@ type ImdsClientInterface interface {
 	GetNetworkContainerInfoFromHost(networkContainerID string, primaryAddress string, authToken string, apiVersion string) (*ContainerVersion, error)
 	GetPrimaryInterfaceInfoFromHost() (*InterfaceInfo, error)
 	GetPrimaryInterfaceInfoFromMemory() (*InterfaceInfo, error)
+	GetNMagentVersion() int
```

Comment:
Rename the function name GetNetworkContainerInfoFromHost and pass all the parameters networkContainerID string, primaryAddress string, apiVersion string)